walk-in-interview/src/main/webapp/new-job/script.js

```diff
@@ -253,46 +229,46 @@ function validateRequiredUserInput() {
   const expiry = document.getElementById('expiry').valueAsNumber;
 
   if (name.value === '') {
-    setErrorMessage(/* msg */ name.placeholder,
-        /** includes default msg */ true);
-    return false;
+      setErrorMessage(/* msg */ name.placeholder,
+          /** includes default msg */ true);
+      return false;
   }
 
   if (description.value === '') {
-    setErrorMessage(/* msg */ description.placeholder,
-        /** includes default msg */ true);
-    return false;
+      setErrorMessage(/* msg */ description.placeholder,
+          /** includes default msg */ true);
+      return false;
   }
 
   if (address.value === '') {
-    setErrorMessage(/* msg */ address.placeholder,
-        /** includes default msg */ true);
-    return false;
+      setErrorMessage(/* msg */ address.placeholder,
+          /** includes default msg */ true);
+      return false;
   }
 
   if (postalCode.value === '') {
-    setErrorMessage(/* msg */ postalCode.placeholder,
-        /** includes default msg */ true);
-    return false;
+      setErrorMessage(/* msg */ postalCode.placeholder,
+          /** includes default msg */ true);
+      return false;
   }
 
   if (payFrequency === '' || Number.isNaN(payMin) || Number.isNaN(payMax) ||
-    payMin > payMax || payMin < 0 || payMax < 0) {
-    setErrorMessage(/* msg */ document.getElementById('pay-title')
-        .textContent, /** includes default msg */ true);
-    return false;
+      payMin > payMax || payMin < 0 || payMax < 0) {
+      setErrorMessage(/* msg */ document.getElementById('pay-title')
+          .textContent, /** includes default msg */ true);
+      return false;
   }
 
   if (duration === '') {
-    setErrorMessage(/* msg */ document.getElementById('duration-title')
-        .textContent, /** includes default msg */ true);
-    return false;
+      setErrorMessage(/* msg */ document.getElementById('duration-title')
+          .textContent, /** includes default msg */ true);
+      return false;
   }
 
   if (Number.isNaN(expiry)) {
-    setErrorMessage(/* msg */ document.getElementById('expiry-title')
-        .textContent, /** includes default msg */ true);
-    return false;
+      setErrorMessage(/* msg */ document.getElementById('expiry-title')
+          .textContent, /** includes default msg */ true);
```

Comment:
FYI annotations look like this: `/* paramName= */true`. You need the `=` and you don't want the double ** which is used for javadoc :) But as i mentioned in the common file you might be able to have default value.